hack/ci/custom-linters/setuplognilerrorcheck/setuplognilerrorcheck.go

```diff
@@ -0,0 +1,83 @@
+package setuplognilerrorcheck
```

Comment:
Nit: rename this package to `analyzers`? That way the next analyzer can just be another file in this package?

Reply:
good idea